apps/prs/src/app/pr-table/columns/target/target.component.ts

```diff
@@ -0,0 +1,44 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {BaseColumn} from '../base';
+
+/** The targets available, and the text to show in the UI. */
+const Targets = [
+  {
+    label: 'target: patch',
+    text: 'Patch',
+  },
+  {
+    label: 'target: minor',
+    text: 'Minor',
+  },
+  {
+    label: 'target: major',
+    text: 'Major',
+  },
+  {
+    label: 'target: lts',
+    text: 'LTS',
+  },
+  {
+    label: 'target: rc',
+    text: 'RC',
+  },
+  {
+    label: 'target: feature',
+    text: 'Feature',
+  },
+];
+
+@Component({
+  selector: 'target-column',
+  templateUrl: './target.component.html',
+  styleUrls: ['./target.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TargetColumn extends BaseColumn {
+  name = 'target';
+
+  getTargetText(target: string) {
+    return Targets.find((t) => t.label === target)?.text || 'No Target';
```

Comment:
I was thinking that we actually preferred the `||` operator because if `text` was empty we wanted to say `No Target`, but now that you pointed this out and I am reading it again, I think `??` might be better for this.